### ISSUE TITLE
First Slide is in the middle

### ISSUE THREAD

**Issue by shakee93** [Wlada/vue-carousel-3d#7]
Thanks for this great carousel.. 

when i implement this the first slide seems to be not showing correctly. what might be the problem ? you can check the fiddle below

https://jsfiddle.net/shakee93/bsofadqa/

![image](https://cloud.githubusercontent.com/assets/14835725/23227720/dd854034-f95f-11e6-8c80-ec1379da991b.png)

**Comment by Wlada** [comment]
you are right, problem exists when you hardcode slides one by one in html. 

with v-for loop everything works fine:
https://jsfiddle.net/Wlada/bq9and8o/

number of slots is doubled when you type very slide in html. 

I will need to see why vue.js doubles number of slides in this case.

**Comment by shakee93** [comment]
Aaah yeah i have also noticed now i am using the v-for. please make a note on the documentation so everyone knows. i was spending hours before ;) 

you could try dynamic scopes or something ? i have no idea about the code though...

thank you for this cool slider though..

![image](https://cloud.githubusercontent.com/assets/14835725/23266202/14001212-fa0d-11e6-92ff-0c9b3ce89ed2.png)

**Comment by Wlada** [comment]
This problem is a bug in Vue.js 2.1.x. I reported a problem on Vue.js issue page
https://github.com/vuejs/vue/issues/4993

**Comment by Wlada** [comment]
Guys from Vue.js dev team confirmed that this is expected behaviour, so it's no a bug. Anyway this helped me to understand some things and how to fix this problem. 

Problem fixed and committed
